packages/coin-avaxc/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@coolwallet/avaxc",
+  "version": "1.0.0",
+  "description": "Coolwallet AVAXC sdk",
+  "main": "lib/index.js",
+  "author": "coolwallet-team",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build:types": "tsc --emitDeclarationOnly",
+    "build:ts": "babel src --out-dir lib --extensions \".ts,.tsx\" --source-maps inline",
+    "build": "npm run build:types && npm run build:ts",
+    "test": "jest --verbose"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.7.0",
+    "@babel/core": "^7.7.0",
+    "@babel/plugin-proposal-class-properties": "^7.7.4",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-typescript": "^7.7.7",
+    "@coolwallet/core": "file:../core",
+    "@types/node": "^14.0.13",
+    "typescript": "^4.4.3"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CoolBitX-Technology/coolwallet3-sdk.git"
+  },
+  "bugs": {
+    "url": "https://github.com/CoolBitX-Technology/coolwallet3-sdk/issues"
+  },
+  "dependencies": {
+    "@types/eth-sig-util": "^2.1.1",
+    "@types/keccak": "^3.0.1",
+    "ajv": "^8.10.0",
+    "bip32": "^2.0.4",
+    "eth-sig-util": "^3.0.1",
+    "keccak": "^3.0.2",
+    "rlp": "^2.2.7"
+  }
```

```suggestion
  },
  "peerDependencies": {
     "@coolwallet/core": "^1.1.5",
  }
```

Should add `@coolwallet/core` as peerDependencies.